cpp/velox/operators/functions/RegistrationAllFunctions.cc

```diff
@@ -67,19 +74,16 @@ void registerFunctionOverwrite() {
   velox::exec::registerFunctionCallToSpecialForm(
       kRowConstructorWithAllNull,
       std::make_unique<RowConstructorWithNullCallToSpecialForm>(kRowConstructorWithAllNull));
-  velox::functions::sparksql::registerBitwiseFunctions("spark_");
   velox::functions::registerBinaryIntegral<velox::functions::CheckedPlusFunction>({"check_add"});
   velox::functions::registerBinaryIntegral<velox::functions::CheckedMinusFunction>({"check_subtract"});
   velox::functions::registerBinaryIntegral<velox::functions::CheckedMultiplyFunction>({"check_multiply"});
   velox::functions::registerBinaryIntegral<velox::functions::CheckedDivideFunction>({"check_divide"});
```

Comment:
It seems some registrations in registerFunctionOverwrite() could be removed, including spark_rand, check_xx. After https://github.com/facebookincubator/velox/commit/372abebc3ded838f2f06544b7c4416db0ec4dab1, we need to adapt to the new names checked_xx.

Reply:
Yes, I also noted this. Will create another pr to fix it.